gst/gst-kvs-plugin/src/GstPlugin.c

```diff
@@ -228,11 +206,6 @@ VOID gst_kvs_plugin_class_init(GstKvsPluginClass* klass)
                                                         DEFAULT_REGION,
                                                         (GParamFlags)(G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS)));
 
-    g_object_class_install_property(gobject_class, PROP_LOG_LEVEL,
-                                    g_param_spec_uint("log-level", "Logging Level", "Logging Verbosity Level", LOG_LEVEL_VERBOSE,
-                                                      LOG_LEVEL_SILENT + 1, LOG_LEVEL_SILENT + 1,
-                                                      (GParamFlags)(G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS)));
-
```

Comment:
Are we sure that we'll not use this logging?

Reply:
Initially setting log-level had no effect, so I removed it. I add it back now and also set the environment variable properly.